src/go/k8s/config/crd/bases/redpanda.vectorized.io_clusters.yaml

```diff
@@ -129,6 +129,11 @@ spec:
               image:
                 description: Image is the fully qualified name of the Redpanda container
                 type: string
+              nodeSelector:
+                additionalProperties:
+                  type: string
+                description: If specified, Redpanda Pod node selectors
```

Comment:
The description is not enough for client. The field is object and additional properties is the type string.

I propose something like:
```suggestion
                description: If specified, Redpanda Pod node selectors. For reference please visit https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node
```